docs/30_getting-started-guide/20_local-development-environment/index.md

```diff
@@ -16,3 +16,8 @@ The following list provides instructions to prepare your local development envir
 3. [Create Development Wallet](30_development-wallet.md): Steps to create development wallet to store key pairs 
 4. [Start Keosd and Nodeos](40_start-nodeos-keosd.md): Steps to start `Keosd` and `Nodeos` 
 5. [Create Development Accounts](50_create-dev-accounts.md): Steps to create you first EOSIO blockchain accounts for smart contracts
+
+## EOSIO Training and Certification Center
+In addition to following the instructions given in the above sections to set up your local development environment, you can also enroll in EOSIO's online learning courses. The following online course provides instructions in a blended format ( textual and videos) to help you set up your local development environment:
```

Comment:
can we avoid the "-ing"?

Reply:
You mean the `ing` In "addition to `following`"?

Reply:
yes

Reply:
seems too wordy to me :

how about being imperative 

"Enroll in EOSIO's online learning courses for interactive guides which show you how to set up your local development environment:"

Reply:
Actually thats bollux ...

Enroll in the EOSIO online learning courses and follow interactive guides which show you how to set up your local development environment:

or something like

Reply:
same comment for the other PR's